zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -78,7 +80,7 @@
 
   private Notebook notebook;
   private NoteManager noteManager;
-  private NotebookRepo notebookRepo;
+  private VFSNotebookRepo notebookRepo;
```

Comment:
Why is this necessary? I think we should work with the interface.

Reply:
Fixed